src/cijobs/project.json

```diff
@@ -15,11 +15,13 @@
     "netcoreapp1.0": {
         "dependencies": {
             "Microsoft.NETCore.App": {
-                "type": "platform",
                 "version": "1.0.0"
             }
         },
         "imports": "dnxcore50"
     }
+  },
+  "runtimes": {
```

Comment:
I don't think we want to check in making this a standalone configuration.  We should probably figure out a way to build them both ways.  (ideas?)